.eslintrc

```diff
@@ -0,0 +1,38 @@
+{
+  "settings": {
+    "import/resolver": {
+      "node": {
+        "paths": ["./src"]
+      }
+    }
+  },
+  "env": {
+    "es6": true,
+    "browser": true,
+    "jest": true
+  },
+  "extends": "@pancakeswap/eslint-config-pancake",
```

Comment:
btw I stole all these rules from pancake but I think they're a good base. we can add/remove any that others have strong opinions for